portfolio/src/main/webapp/style.css

```diff
@@ -56,6 +61,36 @@ main {
   width: 100%;
 }
 
+.color-buttons {
+  margin-top: auto;
+  width: 100%;
+}
+
+.color-buttons i {
+  font-size: 36px;
+  margin: 0 1.5rem;
+}
+
+.color-buttons .coral i {
+  color: var(--coral);
+}
+
+.color-buttons .green i {
+  color: var(--green);
+}
+
+.color-buttons .blue i {
+  color: var(--blue);
+}
+
+.color-buttons .purple i {
+  color: var(--purple);
+}
```

Comment:
I wonder if there's a way to do this that doesn't involve listing each one here - it doesn't seem very DRY. Would it make more sense to just set the color inside the HTML for each button in index.html?